decentraland/unity-renderer
# style: fix textures for avatar modifier/teleport prompt and colapse chat members bar

This PR fixes the missing container on the avatar modifier disclaimer and Teleport Prompt. Besides, the teleport prompt has suffered a structural update in order to align the way scenes' information is displayed in the rest of the platform.

### How to test
1- Open this [link](https://play.decentraland.org/?explorer-branch=style/AvatarModifier-TeleportPrompt-ColapseChatMembers-Fix-Textures)
2- Open any chat conversation and write the coordinates 47,-45 (GolfCraft minigolf scene). Send the message and click on the blue highlighted coords of the message sent. 
- [x] Note that the modal container at the bottom and the skeleton animation UI shapes are rounded. 
- [x] Check the missing pin icon is visible. 
- [x] Once the info loaded, confirm that the structure of the data looks like the following screenshot.

<img width="453" alt="Screenshot 2023-10-30 at 12 37 31" src="https://github.com/decentraland/unity-renderer/assets/51088292/984ec412-760d-4318-9362-d35ac383e73b">

3- Then walk to the "Oh my pixel" interactive wall located inside the scene, parcel 48,-45 to be precise. Get close enough to finally see the avatar modifier warning next to the mini map.
- [x] Check that the warning toast container is rounded instead of a square.

<img width="1091" alt="Screenshot 2023-10-30 at 12 39 36" src="https://github.com/decentraland/unity-renderer/assets/51088292/a2e79cd5-7289-43c0-8911-f548d39430b6">
